src/ui_wizard_config.c

```diff
@@ -1,24 +1,270 @@
+	evas_object_size_hint_weight_set(ic, EVAS_HINT_EXPAND, EVAS_HINT_EXPAND);
+	evas_object_size_hint_align_set(ic, EVAS_HINT_FILL, EVAS_HINT_FILL);
+	elm_table_pack(tb, ic, 1, 1, 1, 1);
+	evas_object_show(ic);
+		
+		
+	lbl = elm_label_add(popup);
+	elm_object_text_set(lbl, "Extern ON");
+	elm_table_pack(tb, lbl, 0, 2, 1, 1);
+	evas_object_show(lbl);
+		
+	ic = elm_icon_add(popup);
+	snprintf(buf, sizeof(buf), "%s/images/left.png", PACKAGE_DATA_DIR);
+	elm_image_file_set(ic, buf, NULL);
+	evas_object_size_hint_min_set(ic, ELM_SCALE_SIZE(321), ELM_SCALE_SIZE(124));
+	evas_object_size_hint_weight_set(ic, EVAS_HINT_EXPAND, EVAS_HINT_EXPAND);
+	evas_object_size_hint_align_set(ic, EVAS_HINT_FILL, EVAS_HINT_FILL);
+	elm_table_pack(tb, ic, 0, 3, 1, 1);
+	evas_object_show(ic);
+		
+	lbl = elm_label_add(popup);
+	elm_object_text_set(lbl, "<color=#ffffff>Intern ON</color>");
+	elm_table_pack(tb, lbl, 1, 2, 1, 1);
+	evas_object_show(lbl);
+		
+	ic = elm_icon_add(popup);
+	snprintf(buf, sizeof(buf), "%s/images/right.png", PACKAGE_DATA_DIR);
+	elm_image_file_set(ic, buf, NULL);
+	evas_object_size_hint_min_set(ic, ELM_SCALE_SIZE(321), ELM_SCALE_SIZE(124));
+	evas_object_size_hint_weight_set(ic, EVAS_HINT_EXPAND, EVAS_HINT_EXPAND);
+	evas_object_size_hint_align_set(ic, EVAS_HINT_FILL, EVAS_HINT_FILL);
+	elm_table_pack(tb, ic, 1, 3, 1, 1);
+	evas_object_show(ic);
+		
+	bt = elm_button_add(popup);
+	elm_object_text_set(bt, "Advanced Settings");
+	elm_table_pack(tb, bt, 0, 4, 2, 1);
+	evas_object_show(bt);
+			
+	evas_object_show(tb);
+	
+	return tb;
+}
+
+
+void
+_page_modes(void *data, Evas_Object *obj EINA_UNUSED, void *event_info EINA_UNUSED)
+{
+	
+   Evas_Object *btn, *content, *ic1, *nf = data;
+//    Elm_Object_Item *it;
+
+   btn = elm_button_add(nf);
+   evas_object_size_hint_align_set(btn, EVAS_HINT_FILL, EVAS_HINT_FILL);
+//    evas_object_smart_callback_add(bt, "clicked", _page3, nf);
+
+   ic1 = elm_icon_add(nf);
+   elm_icon_standard_set(ic1, "arrow_right");
+   evas_object_size_hint_aspect_set(ic1, EVAS_ASPECT_CONTROL_VERTICAL, 1, 1);
+   elm_layout_content_set(btn, "icon", ic1);
+
+   content = _content_modes(nf);
+
+   elm_naviframe_item_push(nf, "Select Mode",
+                                NULL, btn, content,  NULL);
+//    elm_object_item_part_text_set(it, "subtitle", "Here is sub-title part!");
+}
+
+void
+_exit_nf(void *data, Evas_Object *obj EINA_UNUSED, void *event_info EINA_UNUSED)
+{
+	if(popup)
+   {
+        evas_object_del(popup);
+        popup = NULL;
+	}
+}
+
+static void
+_block_clicked_cb(void *data, Evas_Object *obj, void *event_info EINA_UNUSED)
+{
+   if (data)
+     evas_object_del(data);
+   evas_object_del(obj);
+}
+
 
 Evas_Object*
 wizard_config_create(Evas_Object *win)
 {
-   Local_Data *pd = E_NEW(Local_Data, 1);
+	Local_Data *pd = E_NEW(Local_Data, 1);
+
+   if(popup)
+   {
+        evas_object_del(popup);
+        popup = NULL;
+        return;
```

Comment:
This return needs a value, actually, i think the return should not be there at all.